spec/requests/api/v1/users_spec.rb

```diff
@@ -32,4 +32,28 @@ def create_call(user_params)
       it { expect(response).to have_http_status(:created) }
     end
   end
+
+  describe 'PATCH /index' do
+    def update_user_call(user_params)
+      patch api_v1_user_path(user), params: { user: user_params }
+    end
+
+    context 'when is succesful' do
+      let(:user_params) { { email: user.email, password: '123456' } }
+
+      before { update_user_call(user_params) }
+
+      it { expect(response).to have_http_status(:success) }
+    end
+
+    context 'when params are invalid' do
+      let(:user_params) { { email: 'bad_email', password: '123456' } }
+
+      before { update_user_call(user_params) }
+
+      it 'will not update user' do
```

Comment:
Here you are not testing this, you are testing the code not that the user change. Please change the name and add other test that check that the user is not updated